charts/internal/seed-controlplane/charts/ingress-gce/templates/service-account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Is this serviceAccount really needed. Can't we just delete https://github.com/gardener/gardener-extension-provider-gcp/blob/b8608168407cc6f0fab291c16d1b8e63c898e3a4/charts/internal/seed-controlplane/charts/ingress-gce/templates/deployment.yaml#L36?

Reply:
I think, I got it. It's needed as a dummy service account in the seed, while the rbac is needed in the shoot.

Reply:
Still do we really need it in the seed ?